app/ui/browser-australis/model/page-meta.js

```diff
@@ -0,0 +1,28 @@
+/*
+Copyright 2016 Mozilla
+
+Licensed under the Apache License, Version 2.0 (the "License"); you may not use
+this file except in compliance with the License. You may obtain a copy of the
+License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed
+under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+CONDITIONS OF ANY KIND, either express or implied. See the License for the
+specific language governing permissions and limitations under the License.
+*/
+
+import Immutable from 'immutable';
+
+export default Immutable.Record({
+  description: undefined,
+  icon_url: undefined,
+  image_url: undefined,
+  title: undefined,
+  type: undefined,
+  url: undefined,
+  rating: undefined,
+  worst_rating: undefined,
+  best_rating: undefined,
+  review_count: undefined,
+  price: undefined,
+  currency: undefined,
+}, 'PageMeta');
```

Comment:
I didn't know you could add a name to a Record. Does is just set `constructor.name` or is it more directly practical?